gcs_to_bq/main.py

```diff
@@ -0,0 +1,42 @@
+import base64
+import json
+import logging
+from common.census_to_bq import write_state_names_to_bq
+
+
+_HOUSEHOLD_INCOME = 'HOUSEHOLD_INCOME'
+_STATE_NAMES = 'STATE_NAMES'
+
+
+def ingest_bucket_to_bq(event, context):
+  """Main GCF function for moving data from buckets to bigquery.
```

Comment:
Did you configure the cloud function directly in the console? Can we put it in the terraform file? I find it helpful to have all the info of what we have provisioned/configured in one place

Reply:
Once this code is reviewed and pulled, I can roll it into the TF script.

Reply:
Sweet, thanks Brian! I was struggling a bit with understanding terraform so for iteration speed I just manually deployed, but definitely agree putting it in the terraform script is better. In the meantime, would it be helpful for me to put a readme with the different deploy commands I've been using for these?

Reply:
Ah yeah, that would be helpful! I'll make sure to document as much as I can how it's rolled in, and will also go over it during the lightning talk next week too.

Reply:
Added a readme with my deployment setup. Let me know if you need anything else